.github/workflows/deploy.yml

```diff
@@ -0,0 +1,45 @@
+name: deploy
+
+on:
+  push:
+    tags:
+      - 'v*.*.*'
+
+jobs:
+
+  linux-binaries:
+
+    runs-on: ubuntu-latest
+
+    steps:
+    - uses: actions/checkout@v2
+
+    - name: Install stable
+      uses: actions-rs/toolchain@v1
+      with:
+        profile: minimal
+        toolchain: stable
+        override: true
+
+    - name: Build cbindgen
+      run: |
+        cargo build --release
+
+    - name: Strip cbindgen
+      run: |
+        strip target/release/cbindgen
```

Comment:
I'm not sure stripping is great. I'd rather have useful stacks tbh. But this is only for releases so we can change it in a follow-up.